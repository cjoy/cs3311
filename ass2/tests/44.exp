Timothy Cappie-Wood (3363921)
Suggestions for study in 10s1

Course   Title                 UOC Requirement
-------- -----                ---- -----------
COMP1917 Computing 1             6 Level 1 core
MATH1081 Discrete Mathematics    6 Level 1 core
MATH1131 Mathematics 1A          6 Level 1 core
MATH1141 Higher Mathematics 1    6 Level 1 core
Free.... Free Electives (many   48 Free elect
