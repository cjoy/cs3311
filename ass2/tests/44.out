Timothy Cappie-Wood (3363921)
Suggestions for study in 10s1

162 164Course   Title                 UOC Requirement
-------- -----                ---- -----------
